cmd/bblfsh/client.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/bblfsh/sdk/protocol"
 	"google.golang.org/grpc"
+	"srcd.works/go-errors.v0"
+)
+
+var (
+	errProfiler = errors.NewKind("Failed to create a % file at %s")
```

Comment:
I've been reading a bit more about error handling in Golang, and I'm not so sure that variable should be private. I went through the code to look at how we're using errors and to mark some more as private, and also read https://github.com/golang/go/wiki/Errors#naming, and I think that if someone was to use this package externally (which it doesn't make much sense itself, but assuming it would), since this error is returned from a public function, it may make sense for the user to try to check if the error is a `ErrProfiler` for some reason, doesn't it?